logger.py

```diff
@@ -30,6 +31,11 @@
 except KeyError:
     pass
 
+try:
+    log_discord = config['DISCORD']['ENABLED']
```

Comment:
`log_discord` should get initialized to false in the beginning (similar to `log_to_console`), otherwise you will get an NameError if the value does not exist in the config.yaml file, as soon as you try to access it in the if-statement below